pages/password_reset/index.js

```diff
@@ -0,0 +1,36 @@
+import Head from 'components/head';
+import HeroBanner from 'components/HeroBanner/HeroBanner';
+import Content from 'components/Content/Content';
+import PasswordResetForm from 'components/PasswordResetForm/PasswordResetForm';
+import { passwordReset } from 'common/constants/api';
+
+class PasswordReset extends React.Component {
+  state = {
+    success: false,
+  };
+
+  onSuccess = () => {
+    this.setState({ success: true });
+  };
+
+  render() {
+    const { success } = this.state;
+
+    const content = success ? (
+      <p>A confirmation has been sent to the provided email address.</p>
```

Comment:
Alert here as well :) `type="success"`

Reply:
Also, I don't mind the earlier `getContent`, but I'm a big fan of throwing everything into render. With something as simple as a small ternary like this, I'd really prefer it there instead of defined as an extra variable.